### DISCUSSION TITLE
custom shortcuts for custom commands

### DISCUSSION THREAD

**Discussion by eadmaster** [geany/geany#3272] [Ideas]
Allow to set a custom shortcut for every custom command, possibly after checking conflicts.

Related:
https://github.com/geany/geany/issues/2730

**Comment by elextr** [comment]
`Menu->Edit->Preferences->Keybindings` can set keybindings for the custom commands.

**Reply by eadmaster** [reply]
ok, it works fine as intended, however it is limited to 9 commands only and you'll have to remap them if you change the commands order.
